### ISSUE TITLE
DQA crashes from Explore

### ISSUE THREAD

**Issue by kueda** [inaturalist/iNaturalistReactNative#939]
**Describe the bug**
An error gets thrown when you go to the DQA from ObsDetail from Explore (blank white screen).

**To Reproduce**
1. Sign in
2. Go to Explore
3. Tap on an observation to get to ObsDetail
4. Tap the DETAILS tab
5. Tap the VIEW DATA QUALITY ASSESSMENT button
6. App shows a blank white screen or a "Render Error" error screen for dev builds

**Expected behavior**
It should show the DQA. It's actually more important that the DQA be accessible from Explore, since you're usually using it to assess other peoples' observations.

**Screenshots**
![IMG_0032](https://github.com/inaturalist/iNaturalistReactNative/assets/22691/f72f85bc-0e10-40bb-839b-ef51585c0d41)

**Context (please complete the following information):**
 - Device: iPhone SE (3rd gen)
 - OS: iOS 17.1.1
 - App version: [v0.17.1+64](https://github.com/inaturalist/iNaturalistReactNative/releases/tag/v0.17.1%2B64)

**Additional context**
Happening in Android too.

**Comment by joergmlpts** [comment]
![939](https://github.com/inaturalist/iNaturalistReactNative/assets/66563749/9450473f-0ac6-4f89-8291-922c25996709)

The evidence is passed as `[undefined, undefined]`. The caller passes it as

```
               evidence: [observation.observationPhotos, observation.observationSounds],
```

- `observation.observationPhotos` is `undefined`; the list of photos is actually `observation.observation_photos`.
- `observation.observationSounds` does not exist; `observation.sounds` would be the list of sounds but the observation api call does not load them.

**Comment by albullington** [comment]
Closed by PR #988
